rrmngmnt/ssh.py

```diff
@@ -226,19 +249,20 @@ def session(self, timeout=None):
         """
         return RemoteExecutor.Session(self, timeout)
 
-    def run_cmd(self, cmd, input_=None, tcp_timeout=None, io_timeout=None):
+    def run_cmd(self, cmd, input_=None, tcp_timeout=None, io_timeout=None, follow_output=False):
         """
         Args:
             tcp_timeout (float): Tcp timeout
             cmd (list): Command
             input_ (str): Input data
             io_timeout (float): Timeout for data operation (read/write)
+            follow_output (bool): Log each line as soon as it gets to stdout
 
         Returns:
             tuple (int, str, str): Rc, out, err
         """
         with self.session(tcp_timeout) as session:
-            return session.run_cmd(cmd, input_, io_timeout)
+                return session.run_cmd(cmd, input_, io_timeout, follow_output)
```

Comment:
I know about the wrong indent here. Will fix it once I'll be preparing new commit based on reviewers' feedback.